### PR TITLE
Remove jQuery and test without it

### PR DESCRIPTION
**Pull request by cibernox**
🎉 🎉  No jQuery 🎉 🎉

### CODE REVIEW

File: addon/components/one-way-checkbox.js [hunk 1]
```diff
@@ -20,8 +20,9 @@ const OneWayCheckboxComponent = Component.extend(DynamicAttributeBindings, {
     'value'
   ],
 
-  click(event) {
-    invokeAction(this, 'update', this.readDOMAttr('checked'), event);
+  didInsertElement() {
```
Comment (rwjblue) [inline addon/components/one-way-checkbox.js]: Not sure why this needs to change?  The top level `click` method should still be called when not using jQuery (the ember-native-dom-event-dispatcher has tests around this).

Comment (cibernox) [inline addon/components/one-way-checkbox.js]: @rwjblue There's a test that assets that if you call `.stopPropagation` on an action the event doesn't bubble. I started to fail when I changed the dispatcher.

This is is: https://github.com/DockYard/ember-one-way-controls/blob/master/tests/integration/components/one-way-checkbox-test.js#L91-L113

**Comment by cibernox** [comment]
@rwjblue Any idea why the above would fail?

**Comment by cibernox** [comment]
Can anyone cut a new release?

**Pull request merged**